src/util/gas-factory-helpers.ts

```diff
@@ -250,12 +250,8 @@ export async function calculateGasUsed(
   const gasPriceWei = route.gasPriceWei;
   // calculate L2 to L1 security fee if relevant
   let l2toL1FeeInWei = BigNumber.from(0);
-  if ([ChainId.ARBITRUM_ONE, ChainId.ARBITRUM_GOERLI].includes(chainId)) {
-    l2toL1FeeInWei = calculateArbitrumToL1FeeFromCalldata(
```

Comment:
This is exported so idk

Reply:
Instead of deleting `calculateArbitrumToL1FeeFromCalldata`, I think we will want to refactor the same code inside [V3HeuristicGasModelFactory](https://github.com/gzeoneth/smart-order-router/blob/ad63c7fb36c9cbc12a543681124f0a4afa53f642/src/routers/alpha-router/gas-models/v3/v3-heuristic-gas-model.ts#L443-L447).